### DISCUSSION TITLE
How to set http_proxy for Pods of vulnerabilityReport

### DISCUSSION THREAD

**Discussion by liangkun9400** [aquasecurity/trivy-operator#1394] [Help & Support]
Failed to download vulnerability DB from https://ghcr.io. dial tcp: lookup ghcr.io on ...

**Comment by chen-keinan** [comment]
@liangkun9400 can you post a question in our [slack channel](https://slack.aquasec.com/?_ga=2.51428586.2119512742.1655808394-1739877964.1641199050) (#trivy-operator) , maybe someone from community might experience the same issue
